onesixtyone.c

```diff
@@ -200,7 +200,7 @@ void read_hosts(char* filename)
 	FILE* fd;
 	char buf[100];
 	char ch;
-	int c;
+	size_t c;
```

Comment:
I'm not convinced it's strictly necessary, but if I don't have it clang tells me
```
onesixtyone.c:233:9: warning: comparison of integers of different signs: 'int' and 'unsigned long' [-Wsign-compare]
                if (c > sizeof(buf)-1) {
```